src/js/models/list-item.js

```diff
@@ -0,0 +1,51 @@
+import Section from './markup-section';
+import LinkedList from '../utils/linked-list';
+
+export const LIST_ITEM_TYPE = 'list-item';
+
+export default class ListItem extends Section {
+  constructor(tagName, markers=[]) {
+    super(tagName);
```

Comment:
Why not pass `markers` as well and share the logic for the linked list creation and append?